ivadomed/postprocessing.py

```diff
@@ -211,7 +211,7 @@ def label_file_from_coordinates(nifti_image, coord_list):
 
 def threshold_uncertainty(data, uncertainty, thr):
```

Comment:
What about calling it "mask_predictions" or something like this? Or .. to keep this name but rename `threshold_predictions` with `binarize_predictions`? To make clear the difference between the two (below thr = 0 and keep other values VS below=0, above=1).
Also, I am thinking that this function could be used for other purposes than uncertainty, eg #463 --> `mask_predictions(data, data, thr=1e-2)`.